crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -1052,7 +1087,7 @@ mod test {
                 memo: String::new(),
                 rollup_block_number: 1,
                 rollup_return_address: "rollup-defined".to_string(),
-                rollup_transaction_hash: hex::encode([1u8; 32]),
+                rollup_transaction_hash: [1u8; 32].encode_hex_with_prefix(),
```

Comment:
the format here doesn't really matter, it's a string. Also a very seperate concern from the deposits

Reply:
Removed in 1fd6ad2. Thanks for taking a look over!